python/chronos/dev/test/run-pytests-tf1.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cd "`dirname $0`"
+cd ../..
+
+export PYSPARK_PYTHON=python
+export PYSPARK_DRIVER_PYTHON=python
+if [ -z "${OMP_NUM_THREADS}" ]; then
+    export OMP_NUM_THREADS=1
+fi
+
+ray stop -f
+
+echo "Running chronos tests TF1 and Deprecated API"
+python -m pytest -v test/bigdl/chronos/forecaster/test_mtnet_forecaster.py \
+                    test/bigdl/chronos/model/test_mtnet.py \
+                    test/bigdl/chronos/autots/deprecated/ \
+                    test/bigdl/chronos/model/tf1/
```

Comment:
This PR will be merged ahead of #4083 , I don't think we have /model/tf1 at this time

Reply:
Ok, I will change this.